src/sdk.js

```diff
@@ -476,6 +485,20 @@ const endpointDefinitions = [
   },
   { apiName: 'auth', path: 'token', internal: true, method: 'post', interceptors: [] },
   { apiName: 'auth', path: 'revoke', internal: true, method: 'post', interceptors: [] },
+  {
+    apiName: 'auth',
+    path: 'auth_with_idp',
+    internal: false,
```

Comment:
I'm starting to think if this should be an internal method and the public SDK interface method would be called `loginWithIdp` or something of the sort. Would it work better with the current auth methods `login` and `logout`, what do you think?

Reply:
I'm not sure if this would add unnecessary complexity. With `auth.token` it makes sense that the method is internal because it's used by both `login` and `exchangeToken` methods. But on the other hand I think `loginWithIdp? might be clearer and more aligned than `authWithIdp`. I don't have very strong opinion about this.